test/test_jit_fuser_te.py

```diff
@@ -1534,6 +1538,89 @@ def apply(fn):
                     " ".join(["Failed:", str(dtype), op.__name__, device])
                 )
 
+    @unittest.skip("FIXME: fuser doesn't include ListConstruct nodes to the group causing a failure")
+    def test_list_ops(self):
+        def apply(fn):
+            return lambda x, y, z: fn([x * x, y * y, z * z])
```

Comment:
What is needed to make this test work ? There are other cat fusion tests that do work

Reply:
It has something to do with min_group_size=1: if I comment out `torch._C._debug_set_fusion_group_inlining(False)` the test passes. It would be good to fix that logic, but I don't think it's critical.